From: Danny Kukawka <dkukawka@suse.de>
Date: Thu, 5 Jun 2014 00:02:26 +0200
Subject: [PATCH] Rcfiles: remove from runlevel 2

Merge distro patches to git.

This was rcfiles-remove-init-2.patch

(cherry picked from commit ab16e3056e8fc7f26855ca0c58c88ed716443dc1,
amended according to https://build.opensuse.org/request/show/244935)
---
 src/init-radosgw | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/init-radosgw b/src/init-radosgw
index c38ddad..325b877 100644
--- a/src/init-radosgw
+++ b/src/init-radosgw
@@ -3,7 +3,7 @@
 # Provides:          radosgw
 # Required-Start:    $remote_fs $named $network $time
 # Required-Stop:     $remote_fs $named $network $time
-# Default-Start:     2 3 4 5
+# Default-Start:     3 4 5
 # Default-Stop:      0 1 6
 # Short-Description: radosgw RESTful rados gateway
 # Description: radosgw RESTful rados gateway
